src/Symfony/Component/Intl/Locale.php

```diff
@@ -67,21 +67,38 @@ public static function getDefaultFallback()
      */
     public static function getFallback($locale)
     {
-        if (false === $pos = strrpos($locale, '_')) {
-            if (self::$defaultFallback === $locale) {
-                return 'root';
-            }
+        if (function_exists('locale_parse')) {
```

Comment:
this looks weird to me. This function is part of ext-intl, so the polyfill will never be used when this function exists

Reply:
That's the point. did I miss something?

Reply:
ah, I missed the fact that this is not a class from the polyfill. so fine